spec/dependent_styles_spec.rb

```diff
@@ -1,82 +1,79 @@
-Dependents.each_pair do |id, (filename, path, style, reason)|
+Dependents.each_pair do |basename, (filename, path, style, reason)|
 
-  describe "dependent style #{id}" do
+  describe "#{basename} (dependent):" do
```

Comment:
Agree that the colon isn't necessary. To stay as brief as possible, we could do:

```
apa must
dependent/nature-biotechnology must
```